### PR TITLE
Remove Paas app

### PR DESCRIPTION
**Pull request by bj2k61**
## Trello card URL
https://trello.com/c/qRuJfuKh/272-decommission-paas-environments)

## Changes in this PR:
Removes the Paas APP

### CODE REVIEW

File: terraform/app/modules/paas/variables.tf [hunk 1]
```diff
@@ -7,23 +7,12 @@ variable "app_docker_image" {
...
 variable "docker_username" {
 }
 
 variable "docker_password" {
 }
 
-variable "logging_service_binding_enable" {
-}
-
 variable "logging_url" {
```
Comment (saliceti) [inline terraform/app/modules/paas/variables.tf:16]: is this used?

File: terraform/app/modules/paas/variables.tf [hunk 2]
```diff
@@ -182,9 +151,6 @@ locals {
   ]
 
   postgres_instance_service_key = { DATABASE_URL = cloudfoundry_service_key.postgres_instance_service_key.credentials.uri }
-
-  app_user_provided_service_bindings = var.logging_service_binding_enable ? [cloudfoundry_user_provided_service.logging.id] : []
-  app_service_bindings               = concat(local.app_cloudfoundry_service_instances, local.app_user_provided_service_bindings)
   logging_service_name               = "${var.service_name}-logging-${var.environment}"
```
Comment (saliceti) [inline terraform/app/modules/paas/variables.tf:154]: is this used?

File: terraform/app/modules/paas/main.tf
Comment (saliceti) [inline terraform/app/modules/paas/main.tf:1]: Can we remove the cloudfoundry_user_provided_service as it's only used by the app?

File: terraform/app/modules/paas/variables.tf [hunk 3]
```diff
@@ -7,23 +7,12 @@ variable "app_docker_image" {
...
-}
-
-variable "app_stopped" {
-  default = false
-}
-
 variable "docker_username" {
 }
 
 variable "docker_password" {
```
Comment (saliceti) [inline terraform/app/modules/paas/variables.tf:13]: is this used?

File: terraform/workspace-variables/qa.tfvars.json [hunk 4]
```diff
@@ -27,17 +27,10 @@
   ,
   "documents_s3_bucket_force_destroy": false,
   "environment": "qa",
-  "paas_app_start_timeout": "180",
```
Comment (saliceti) [inline terraform/workspace-variables/qa.tfvars.json:30]: Can we remove these variables in all environments?

File: terraform/app/modules/paas/variables.tf (hunk 1 above)
Comment (bj2k61) [reply terraform/app/modules/paas/variables.tf:16]: resolved

**Comment by bj2k61** [comment]
[Decommission PaaS environments](https://trello.com/c/qRuJfuKh/272-decommission-paas-environments)

**Review by saliceti — APPROVED** [review]

**Pull request merged**